### PR TITLE
changes to support postgres uri - this fixes #173

### LINKED ISSUE #173: Postgres connection uri wrong
Says mysql instead of postgres!

### PR DESCRIPTION
**Pull request by ccemeraldeyes**

### CODE REVIEW

File: brokerapi/brokers/models/db.go [hunk 1]
```diff
@@ -33,6 +34,14 @@ type ServiceBindingCredentials struct {
 	BindingId         string
 }
 
+func (sbc ServiceBindingCredentials) GetOtherDetails() map[string]string {
+	var creds map[string]string
+	if err := json.Unmarshal([]byte(sbc.OtherDetails), &creds); err != nil {
```
Comment (johnsonj) [inline brokerapi/brokers/models/db.go]: any chance this one is empty?

File: integration_tests/async_integration_test.go [hunk 2]
```diff
@@ -242,6 +242,7 @@ var _ = Describe("AsyncIntegrationTests", func() {
 			Expect(credsMap["Username"]).ToNot(Equal(""))
 			_, err = sslService.Get(gcpBroker.RootGCPCredentials.ProjectId, cloudsqlInstanceName, credsMap["Sha1Fingerprint"]).Do()
 			Expect(err).NotTo(HaveOccurred())
+			Expect(credsMap["uri"]).To(ContainSubstring("postgres"))
```
Comment (johnsonj) [inline integration_tests/async_integration_test.go]: add this for MySQL as well

File: brokerapi/brokers/account_managers/sql_account_manager.go [hunk 3]
```diff
@@ -165,11 +165,29 @@ func (sam *SqlAccountManager) pollOperationUntilDone(op *googlecloudsql.Operatio
...
+	instanceDetails := instanceRecord.GetOtherDetails()
+	bindDetails := bindRecord.GetOtherDetails()
+
+	sid := instanceRecord.ServiceId
+	service_to_name, err := utils.MapServiceIdToName()
+	if err != nil {
+		return map[string]string{}, err
+	}
+
 	combinedCreds := utils.MergeStringMaps(bindDetails, instanceDetails)
```
Comment (johnsonj) [inline brokerapi/brokers/account_managers/sql_account_manager.go]: nit: move up to 171, code introduced at 172/176 splits the logical flow

File: brokerapi/brokers/models/db.go (hunk 1 above)
Comment (mkjelland) [reply brokerapi/brokers/models/db.go]: Can OtherDetails be anything? Or is is guaranteed to be creds?

File: brokerapi/brokers/account_managers/sql_account_manager.go (hunk 3 above)
Comment (ccemeraldeyes) [reply brokerapi/brokers/account_managers/sql_account_manager.go]: ack

File: brokerapi/brokers/models/db.go (hunk 1 above)
Comment (ccemeraldeyes) [reply brokerapi/brokers/models/db.go]: This should always contain stuff and should panic if it doesn't.

File: integration_tests/async_integration_test.go (hunk 2 above)
Comment (ccemeraldeyes) [reply integration_tests/async_integration_test.go]: ack

**Pull request merged**